src/ripple/app/consensus/RCLConsensus.cpp

```diff
@@ -843,7 +843,8 @@ RCLConsensus::validate(RCLCxLedger const& ledger, bool proposing)
     v->setTrusted();
     // suppress it if we receive it - FIXME: wrong suppression
     app_.getHashRouter().addSuppression(signingHash);
-    app_.getValidations().addValidation(v, "local");
+    if (! app_.getValidations().addValidation(v, "local"))
```

Comment:
Or even simpler, just make it the condition for validating when you start the round https://github.com/mtrippled/rippled/blob/22536534936b9508f8b4dfc54d5abae869579e8e/src/ripple/app/consensus/RCLConsensus.cpp#L917